src/main/java/uk/gov/pay/directdebit/gatewayaccounts/api/GatewayAccountResponse.java

```diff
@@ -33,16 +35,25 @@
     private String type;
     @JsonProperty("analytics_id")
     private String analyticsId;
+    @JsonProperty("is_linked")
+    private final boolean isLinked;
```

Comment:
I wonder if `is_connected` might be better for this? GoCardless refer to them as connected accounts (and I think Stripe do something similar and consistency is always nice).

Reply:
Agreed. I've renamed this.